feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -157,8 +157,8 @@ class BillingWrapper(
         log(LogIntent.DEBUG, OfferingStrings.FETCHING_PRODUCTS.format(productIds.joinToString()))
         executeRequestOnUIThread { connectionError ->
             if (connectionError == null) {
-                val googleType = productType.toGoogleProductType() ?: BillingClient.ProductType.SUBS
```

Comment:
This was making a test fail and figured we should keep it as it was

Reply:
you mean this is how it was pre-bc5?

Reply:
yes @beylmk